InAppMessaging/Example/Tests/FIRIAMClearcutUploaderTests.m

```diff
@@ -51,20 +66,24 @@ - (void)setUp {
                                                                       batchSendSize:10];
 
   self.mockUserDefaults = OCMClassMock(NSUserDefaults.class);
+  self.cachePath = [self generatedCachePath];
   OCMStub([self.mockUserDefaults integerForKey:[OCMArg any]]).andReturn(0);
 }
 
 - (void)tearDown {
-  // Put teardown code here. This method is called after the invocation of each test method in the
-  // class.
+  [[NSFileManager defaultManager] removeItemAtPath:self.cachePath error:NULL];
   [super tearDown];
 }
 
 - (void)testUploadTriggeredWhenWaitTimeConditionSatisfied {
   // using a real storage in this case
   FIRIAMClearcutLogStorage *logStorage =
       [[FIRIAMClearcutLogStorage alloc] initWithExpireAfterInSeconds:1000
-                                                     withTimeFetcher:self.mockTimeFetcher];
+                                                     withTimeFetcher:self.mockTimeFetcher
+                                                       withCacheFile:self.cachePath];
+
+  NSTimeInterval currentMoment = 10000;
+  OCMStub([self.mockTimeFetcher currentTimestampInSeconds]).andReturn(currentMoment);
```

Comment:
I think we'd still want this to go above the `logStorage` instantiation since it uses `mockTimeFetcher`.

Reply:
As well as all the other instances of this happening below.